File: MediaLibrary/MediaLibraryWebUI/Models/Configurations/PlaylistConfiguration.cs
using MediaLibraryWebUI.Models.Interfaces;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using static MediaLibraryWebUI.UIEnums;

namespace MediaLibraryWebUI.Models.Configurations
{
    public class PlaylistConfiguration : BaseConfiguration
    {
        public PlaylistConfiguration()
        {
        }

        public int SelectedPlaylistId { get; set; }
        public PlaylistPages SelectedPlaylistPage { get; set; }
        public PlaylistSort SelectedPlaylistSort { get; set; }
        public PlaylistTabs SelectedPlaylistTab { get; set; }
    }
}

File: MediaLibrary/MediaLibraryWebUI/lib/app/configurations/playlist-configuration.ts
import BaseConfiguration from './base-configuration'
import * as Enums from '../assets/enums'

export default class PlaylistConfiguration extends BaseConfiguration {
    private selectedPlaylistId: number;
    private selectedPlaylistPage: Enums.PlaylistPages;
    private selectedPlaylistSort: Enums.PlaylistSort;

    constructor() {
        super();
    }
}
